index.html

```diff
@@ -247,6 +247,8 @@ <h4 class="popup__type">Квартира</h4>
       <button class="error__button" href="#">Попробовать снова</button>
     </div>
   </template>
+<script src="js/main.js">
```

Comment:
Так кончно можно, но лучше в одну строку
```suggestion
<script src="js/main.js"></script>
```